pallets/assets-info/src/tests.rs

```diff
@@ -20,7 +20,7 @@ fn set_info_and_retrieve_works_ok() {
             info.name.clone(),
             info.symbol.clone(),
             info.description.clone(),
-            info.decimals.clone(),
+            info.decimals,
```

Comment:
why decimals are treated differently compared to the rest fields of the info struct?